### PR TITLE
adds appveyor config

### PR DESCRIPTION
**Pull request by mathiask88**
@ralphtheninja

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# http://www.appveyor.com/docs/appveyor-yml
+
+# Don't actually build
+build: off
```
Comment (ralphtheninja) [inline appveyor.yml]: What does this mean?

Comment (mathiask88) [reply appveyor.yml]: I think this is a MSBuild thing. Node.js appveyor docs includes this line https://www.appveyor.com/docs/lang/nodejs-iojs/#quick-start

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # prebuild-install
 
 [![Build Status](https://travis-ci.org/prebuild/prebuild-install.svg?branch=master)](https://travis-ci.org/prebuild/prebuild-install)
+[![Build status](https://ci.appveyor.com/api/projects/status/6v6hxxwgjrr99pc8/branch/master?svg=true)](https://ci.appveyor.com/project/mathiask88/prebuild-install)
```
Comment (ralphtheninja) [inline README.md]: Possible to get this under `prebuild/prebuild-install`? No biggie though.

Comment (mathiask88) [reply README.md]: Is this possible with my appveyor account? I get `Project not found or access denied.` Do we have a `prebuild` appveyor account?

Comment (ralphtheninja) [reply README.md]: Nevermind, lets merge this now and tweak stuff later.

Comment (vweevers) [reply README.md]: We could take the same approach as in Level: a github machine user (`prebuild-ci`) as main AppVeyor account, and subaccounts for maintainers.

**Pull request merged**